website/content/en/highlights/2021-12-14-improved-concurrency-model.md

```diff
@@ -0,0 +1,34 @@
+---
+date: "2021-12-14"
+title: "Improved concurrency model"
+description: "New parallel processing with improved performance"
+authors: ["barieom"]
+pr_numbers: [10265]
+release: "0.19.0"
+hide_on_release_notes: false
+badges:
+  type: announcement
+---
+
+We've released an improved concurrency model that provides measurable performance
+improvements and enables more efficient vertical scaling.
+
+Previously, Vector set the limit to active tasks to the number of available CPUs
+in a given environment. Transforms were run sequentially, which made transforms
+a bottleneck in Vector pipelines.
```

Comment:
```suggestion
Previously, Vector was limited to executing transforms on a single thread, which led to them often being the bottleneck in Vector pipelines.
```

Reply:
I would say something like `limited to using a single thread per transform`.